src/controllers/seller controller/sellerController.ts

```diff
@@ -1,7 +1,11 @@
-import { Request, Response } from 'express';
+import { Request, Response, response } from 'express';
 import jwt, { JwtPayload } from 'jsonwebtoken';
 import { SellerService } from '../../services/seller.Service.js';
```

Comment:
Move all service exports to a single file so that they can call be imported at once